apps/root/webpack.common.js

```diff
@@ -91,6 +45,7 @@ module.exports = {
       'process.env.ENABLED_TRANSLATIONS': JSON.stringify(process.env.ENABLED_TRANSLATIONS),
     }),
     new WebpackBar(),
+    new ForkTsCheckerWebpackPlugin(),
```

Comment:
<img width="905" alt="Screenshot 2024-07-10 at 10 49 11 AM" src="https://github.com/Balmy-protocol/dca-fe/assets/94012134/ec6f7530-ed2e-46ed-a169-e1354ea02da4">
Should we modify this so it can use a bit more memory?